python/gpusim_server.py

```diff
@@ -229,6 +229,8 @@ def parse_args():
                         help="Start HTTP server for debugging, not secure enough for production machine") #noqa
     parser.add_argument('--cpu_only', action='store_true',
                         help="Search the database on the CPU, not the GPU (slow)") #noqa
+    parser.add_argument('--gpu_bitcount', default='0',
+                        help="Provide the maximum bitcount for fingerprints on GPU") #noqa
```

Comment:
can you add validation to make this and --cpu_only mutually exclusive?

Reply:
Yeah, I'm going to delay fixing this until I switch to the other command line handler.